### PR TITLE
test: try to avoid race test

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs520 <crazycs520@gmail.com>

### What problem does this PR solve?

Temporarily avoid the race problem in https://github.com/pingcap/tidb/issues/18526, but this PR doesn't fix the root cause of the race.

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- No code

Side effects

- No

### Release note 

- No

### CODE REVIEW

**Comment by crazycs520** [comment]
Yes, `config.GetGlobalConfig().EnableCollectExecutionInfo = false` should avoid the `Read` in the race.

File: planner/cascades/integration_test.go [hunk 1]
```diff
@@ -33,6 +34,8 @@ type testIntegrationSuite struct {
 }
 
 func newStoreWithBootstrap() (kv.Storage, error) {
+	// TODO: remove this and find out the root cause of race.
```
Comment (XuHuaiyu) [inline planner/cascades/integration_test.go:37]: This comment is hard to understand, it's better to add the issue link here

Comment (crazycs520) [reply planner/cascades/integration_test.go:37]: done.

**Pull request closed without merging**